desktop_app/start_application.py

```diff
@@ -2,8 +2,8 @@
 import sys
 from PyQt5.QtWidgets import QApplication
 
-sys.path.insert(0, 'GUI/Py')
-import ui # noqa
+sys.path.insert(0, '..')
```

Comment:
I don't like the solution when we're adding to PYTHONPATH parent directory. I'd rather add `desktop_app/GUI/Py` to PYTHONPATH in tests file.